assets/src/scripts/components/hydrograph/scales.js

```diff
@@ -5,7 +5,7 @@ const { createSelector } = require('reselect');
 
 const { default: scaleSymlog } = require('../../lib/symlog');
 const { layoutSelector, MARGIN } = require('./layout');
-const { flatPointsSelector, timeSeriesSelector, variablesSelector, visiblePointsSelector, currentVariableSelector } = require('./timeseries');
+const { flatPointsSelector, timeSeriesSelector, variablesSelector, visiblePointsSelector, currentVariableSelector, requestTimeRangeSelector } = require('./timeseries');
```

Comment:
flatPointsSelector is no longer used in scales.js

Reply:
👍 Thanks - my eslint plugin has stopped working.